minigui/board.ts

```diff
@@ -188,11 +186,11 @@ class Board extends View {
 
     ctx.fillStyle = this.stoneFill(color, alpha);
 
-    let r = 0.48 * this.stoneRadius;
+    let r = this.stoneRadius;
     for (let p of ps) {
       let c = this.boardToCanvas(p.row, p.col);
       ctx.beginPath();
-      ctx.translate(c.x, c.y);
+      ctx.translate(c.x + 0.5, c.y + 0.5);
```

Comment:
The 0.5 offsets the center of the circle by half a pixel so that it's correctly centered.

I actually need to do another pass over the all the drawing code to make it consistent about how it handles issues like this.